todo-frontend/src/components/Create.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+    <div>
+        <p>Title: <input v-model="title" type="text"></p>
+        <p>Description: <textarea v-model="description" rows="4" cols="50"></textarea></p>
+        <p>Priority: 
+            <select v-model= "priority">
+                <option selected="selected" value="LOW">LOW</option>
+                <option value="MEDIUM">MEDIUM</option>
+                <option value="HIGH">HIGH</option>
+            </select>
+        </p>
+
+        
+
+
+        <button @click="submit()">Submit</button>
+    </div>
+</template>
+
+<script>
+import { mapActions } from 'vuex';
+export default {
+    data () {
+        return {
```

Comment:
ubaci title desc i prio u neki form objekat, pa onda mozes dole pozivas samo this.addTodo(form)